catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/feedback/FeedbackRequest.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.feedback;
+
+public class FeedbackRequest {
```

Comment:
Some of the methods in this class are self-explanatory, but some are not, like getQuery. What does the return value for getQuery mean? Some javadoc on those methods would be nice.